src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,124 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CalculatorTest {
+
+    @Test
+    public void additionNegativeOk() {
```

Comment:
```suggestion
    public void addNegativeNumberOk() {
```
аналогічно з рештою)

Reply:
done